editoast/src/core/stdcm.rs

```diff
@@ -115,28 +113,24 @@ pub struct UndirectedTrackRange {
     pub end: u64,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, ToSchema)]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
 #[serde(tag = "status", rename_all = "snake_case")]
 // We accepted the difference of memory size taken by variants
 // Since there is only on success and others are error cases
 #[allow(clippy::large_enum_variant)]
-pub enum STDCMResponse {
+pub enum STDCMCoreResponse {
```

Comment:
I'm not sure that I understand the renaming. This `struct` is defined inside a module name `core` so we already know it's relative to core's logic.

Reply:
I renamed it to avoid conflicts and `as`, since we have another `STDCMResponse` in `views`, and both are used in the same file.